src/main/java/org/cyclonedx/generators/xml/AbstractBomXmlGenerator.java

```diff
@@ -495,7 +122,7 @@ public String toXmlString() throws TransformerException {
     public String toString() {
```

Comment:
minor: Above, on line 116, the javadoc says the method will return null if an exception occurs. Should probably change:
`return null if {@link #toXmlString()}`
to:
`return an empty string if {@link #toXmlString()}`

Reply:
In: 682e00f